catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/handlers/CqlTransformHandler.java

```diff
@@ -217,4 +236,46 @@ private void attachFileToResponse(
         "Successfully output file using transformer id {}",
         queryResponseTransformer.getProperty("id"));
   }
+
+  private Map<String, Serializable> cswTransformArgumentsAdapter() {
+    Map<String, Serializable> args = new HashMap<>();
+    args.put(CswConstants.IS_BY_ID_QUERY, true);
+    return args;
+  }
+
+  private Map<String, Serializable> csvTransformArgumentsAdapter(
+      Map<String, Serializable> arguments) {
+    String columnOrder = "\"columnOrder\":" + (String) mapper.toJson(arguments.get("columnOrder"));
```

Comment:
✏️ Might be worth it to extract the various string keys into static variable since they're used multiple times.